lib/vdsm/tool/configurators/multipath.py

```diff
@@ -214,6 +214,7 @@
 # We blacklist following protocols:
 # scsi:adt  local storage protocol
 # scsi:sbp  local storage protocol
+# RBD devices were also added to the multipath blacklist (BZ#1881832)
```

Comment:
This is a list of protocols we blacklist. This is not the place to add
the note about RBD.

Reply:
ACK, looked for the right place for that :)